packages/synapse-react-client/src/components/DownloadCart/MeetAccessRequirementCard.tsx

```diff
@@ -42,12 +46,22 @@ export const MeetAccessRequirementCard: React.FunctionComponent<
   let iconType = ''
   let description = ''
 
+  const arFriendlyName = useMemo(() => {
+    if (!ar) {
+      return ''
+    } else if (ar.name.match(/^\d+$/)) {
+      // Don't show the AR name to the user if it's just the numeric ID
+      return ''
+    }
+    return ar.name
+  }, [ar])
```

Comment:
Small change, I know governance has a lot of ARs that are just IDs. Showing them on this card to the user attempting to get access is probably more confusing than helpful.